spec/models/post_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'testing title', text: 'hello from rspec') }
+
+  it 'should have a title' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'should have content' do
+    subject.text = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments counter should be an integer' do
+    subject.comments_counter = 'some string'
+    expect(subject).to_not be_valid
+  end
+
+  it 'likes counter should be an integer' do
+    subject.likes_counter = 'some string'
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] @Xghayor Great work adding tests for the post model validations, however, you forgot to add tests for the custom method `five_recent_comments` which is part of the requirements
![image](https://github.com/Xghayor/social-media/assets/61408860/1078a536-70eb-4c88-8c7a-3c9e58e800c8)